RandomData.Api/StringGeneration/ServiceImplementations/WordsStringGenerationService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using RandomData.Api.Services.FileReader;
+using RandomData.Api.Services.Random;
+using RandomData.Api.StringGeneration.Dto;
+using RandomData.Api.StringGeneration.Exceptions;
+using RandomData.Api.StringGeneration.Extensions;
+using RandomData.Api.StringGeneration.Validators;
+
+namespace RandomData.Api.StringGeneration.ServiceImplementations
+{
+    public class WordsStringGenerationService : IStringGenerationService
+    {
+        private readonly IRandom _random;
+        private readonly GetStringParametersValidator _validator = new GetStringParametersValidator();
+        private readonly IEnumerable<string> _words;
```

Comment:
You can use Array or even ISet/HashSet. More specific type gives extra information about usage ad purpose.